e2e_tests/standard/test_copy.py

```diff
@@ -15,53 +15,42 @@
 #
 import random
 import pytest
-from faker import Faker
+import itertools
+from neptune.new.metadata_containers import MetadataContainer
 
-import neptune.new as neptune
-from neptune.new.run import Run
-from neptune.new.project import Project
+from e2e_tests.base import BaseE2ETest, AVAILABLE_CONTAINERS, fake
 
-from e2e_tests.base import BaseE2ETest
-
-fake = Faker()
+# List of every possible container type pair for instance: "run-run, run-model, model-model-version, ..."
+ALL_CONTAINERS_PAIRS = list(
+    map(
+        lambda containers_pair: "-".join(containers_pair),
```

Comment:
see my comment at `e2e_tests/conftest.py`, I don't think this join-split is necessary

Reply:
Answered in another thread